lib/Widget/evaluationForms/post-hos/post-hos-day0/recovery_readiness_form.dart

```diff
@@ -0,0 +1,376 @@
+import 'package:AbdoCare_Web/Widget/shared/alert_style.dart';
+import 'package:AbdoCare_Web/services/interfaces/firebase_service_interface.dart';
+import 'package:AbdoCare_Web/services/service_locator.dart';
+import 'package:flutter/material.dart';
+
+class RecoveryReadinessForm extends StatefulWidget {
+  final String hn;
+  RecoveryReadinessForm({Key key, @required this.hn}) : super(key: key);
+
+  @override
+  _RecoveryReadinessFormState createState() => _RecoveryReadinessFormState();
+}
+
+class _RecoveryReadinessFormState extends State<RecoveryReadinessForm> {
+  IFirebaseService _firebaseService = locator<IFirebaseService>();
+  var _knowFullName = false;
+  var _knowTime = false;
+  var _knowPlace = false;
+  var _fever = false;
+  var _breathGasp = false;
+  var _palpitations = false;
+  var _dizzy = false;
+  var _vomit = false;
+  var _readyRecover = false;
+  var patientState;
+  var _getpatientState;
+  @override
+  void initState() {
+    super.initState();
+    initData();
+  }
+
+  void initData() async {
+    patientState = await _firebaseService.getPatientState(hn: widget.hn);
+    setState(() {
+      _getpatientState = patientState;
+    });
+  }
+
+  bool completeAllCheckBox() {
+    if (_knowFullName &&
+        _knowTime &&
+        _knowPlace &&
+        _fever &&
+        _breathGasp &&
+        _palpitations &&
+        _dizzy &&
+        _vomit == true) {
+      return true;
+    } else {
+      return false;
```

Comment:
What are these lines for ?

Reply:
<img width="501" alt="Screen Shot 2564-04-26 at 20 28 32" src="https://user-images.githubusercontent.com/57941667/116090364-fb539f00-a6cd-11eb-846a-d9244e6adf6e.png">
เอาไว้สำหรับเช็คว่า user ติ๊กหมดไหมถ้าติ๊กหมด ช่องผู้ป่วยรู็สึกตัวดีจะขึ้นเอง

Reply:
Uploading Screen Recording 2564-04-26 at 20.46.00.mov…